docs/how-to/solve-with-imitation-dynamics.rst

```diff
@@ -0,0 +1,40 @@
+.. _how-to-use-imitation-dynamics:
+
+Solve with Imitation Dynamics
+==============================
+
+One of the algorithms implemented in :code:`Nashpy` is called
+:code:`imitation_dynamics()`, this is implemented as a method on the :code:`Game`
+class::
+
+    >>> import nashpy as nash
+    >>> import numpy as np
+    >>> A = np.array([[3, -1], [-1, 3]])
+    >>> B = np.array([[-3, 1], [1, -3]])
+    >>> rps = nash.Game(A,B)
+
+This :code:`imitation_dynamics` method returns a generator of all the
+equilibria::
+
+    >>> ne_imitation_dynamics = rps.imitation_dynamics()
+    >>> print(list(ne_imitation_dynamics))
+    [(array([0., 1.]), array([1., 0.]))]
+
+Note:
+Even with a zero-sum game, the probability of strategies to achieve Nash Equilibrium is not evenly distributed; this is because the system is not random but runs only on imitating the opposition's strategy.
```

Comment:
I don't understand what this sentence is saying.

Reply:
I am under the impression that if there is a zero-sum game, we will have an equilibrium strategy with equal distribution of the choices to achieve Nash equilibrium. for instance where A=[[1,-1][-1,1]] and B = -A the outcome will be [0.5,0.5] for A and  [0.5,0.5] for B . This may not be the case in imitation because the iteration is done on the last best choice made by the opponent. In the example mentioned in the documentation, the results were [(array([0., 1.]), array([1., 0.]))] even though it was a zero-sum game. Hence the Note.

Reply:
That's not correct. Indeed if it was game theory would be a very simple subject. If you look through the documentation you'll find another examples of games where this is not true.